hack/generator/pkg/config/type_transformer.go

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT license.
+ */
+
+package config
+
+import (
+	"fmt"
+	"github.com/Azure/k8s-infra/hack/generator/pkg/astmodel"
+)
+
+type TransformTarget struct {
+	PackagePath string `yaml:",omitempty"`
+	Name        string `yaml:",omitempty"`
+}
+
+// A TypeTransformer is used to remap types
+type TypeTransformer struct {
+	TypeMatcher `yaml:",inline"`
+
+	Transform TransformTarget // TODO: Don't love this property name or type name...
```

Comment:
I'd use `Target` here - it's more specific than `Transform` (which is implied by the context anyway) and I think it reads better where it's used.

Reply:
Done